lib/Module/KModule.cpp

```diff
@@ -275,63 +285,36 @@ void KModule::prepare(const Interpreter::ModuleOptions &opts,
   if (!operandTypeCheckPass->checkPassed()) {
     klee_error("Unexpected instruction operand types detected");
   }
+}
 
+void KModule::manifest(InterpreterHandler *ih, bool forceSourceOutput) {
   // Write out the .ll assembly file. We truncate long lines to work
   // around a kcachegrind parsing bug (it puts them on new lines), so
   // that source browsing works.
```

Comment:
Please make sure to remove the part of the comment referring to the workaround, now that it's gone (or even the whole comment).

Reply:
Good point. Thanks!